jqm-all/jqm-wstst/src/test/java/com/enioka/jqm/ws/api/ServiceSimpleStandaloneTest.java

```diff
@@ -0,0 +1,50 @@
+package com.enioka.jqm.ws.api;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ServiceSimpleStandaloneTest {
+
+    @Test
+    public void testFullIpToSequence() {
+        final var ip = "192.168.124.234";
+        final var expected = 192_168_124_234_000_000L;
```

Comment:
Syntactic sugar the underscore ?

Reply:
Yes, it makes large numbers more readable